build.sbt

```diff
@@ -7,11 +7,6 @@ scalaVersion := "2.12.3"
 sbtVersion in Global := "1.0.0"
 crossSbtVersions := Seq("1.0.0", "0.13.16")
 
-scalaCompilerBridgeSource := {
```

Comment:
Just for my own understanding: do you know why this is no longer required?

Reply:
This was a fix for cross compiling in sbt 0.13. sbt 1.0 no longer needs it so it can be removed (read: needs to be removed otherwise you'll get error messages that the internet tells you will be fixed if you **_add_** this section to your build.sbt)